docs/source/guide/mlbackend.md

```diff
@@ -0,0 +1,98 @@
+
+
+After you define the loaders, you can define two methods for your model: an inference call and a training call. 
+
+### Inference call
+
+Use an inference call to get pre-annotations from your model on-the-fly. You must update the existing predict method in the example ML backend scripts to make them work for your specific use case. 
+
+Write your own code to override the `predict(tasks, **kwargs)` method, which takes [JSON-formatted Label Studio tasks](tasks.html#Basic-Label-Studio-JSON-format) and returns predictions in the [format accepted by Label Studio](predictions.html).
+
+**Example**
+
+This example defines an inference call that pulls the labeling configuration schema and then outputs the predictions from your model in that format so that Label Studio can understand and display the predictions in the Label Studio UI. This example uses a labeling configuration that uses the [`Choices` tag](/tags/choices.html). 
+
+```python
+def predict(self, tasks, **kwargs):
+    predictions = []
+    # Get annotation tag first, and extract from_name/to_name keys from the labeling config to make predictions
+    from_name, schema = list(self.parsed_label_config.items())[0]
+    to_name = schema['to_name'][0]
+    for task in tasks:
+        # for each task, return classification results in the form of "choices" pre-annotations
+        predictions.append({
+            'result': [{
+                'from_name': from_name,
+                'to_name': to_name,
+                'type': 'choices',
+                'value': {'choices': ['My Label']}
+            }],
+            # optionally you can include prediction scores that you can use to sort the tasks and do active learning
+            'score': 0.987
+        })
+    return predictions
+```
+
+
+### Training call
+Use the training call to update your model with new annotations. You don't need to use this call in your code, for example if you just want to pre-annotate tasks without retraining the model. If you do want to retrain the model based on annotations from Label Studio, use this method. 
+
+Write your own code to override the `fit(annotations, **kwargs)` method, which takes [JSON-formatted Label Studio annotations](https://labelstud.io/guide/export.html#Raw-JSON-format-of-completed-labeled-tasks) and returns an arbitrary dict where some information about the created model can be stored.
+
+**Example**
+```python
+def fit(self, completions, workdir=None, **kwargs):
+    # ... do some heavy computations, get your model and store checkpoints and resources
+    return {'checkpoints': 'my/model/checkpoints'}  # <-- you can retrieve this dict as self.train_output in the subsequent calls
+```
+
+After you wrap your model code with the class, define the loaders, and define the methods, you're ready to run your model as an ML backend with Label Studio. See the [Quickstart](ml.html#Quickstart). 
+
+## ML backend tutorials
```

Comment:
Let's move it on separate page? Looks like it's important and often used section

Reply:
This whole page is about tutorials so I'm going to leave this here